Tools/WebKitTestRunner/TestController.cpp

```diff
@@ -1085,6 +1102,9 @@ bool TestController::resetStateToConsistentValues(const TestOptions& options, Re
     // Reset notification permissions
     m_webNotificationProvider.reset();
     m_notificationOriginsToDenyOnPrompt.clear();
+#if ENABLE(NOTIFICATIONS)
```

Comment:
No need for this since this SPI is not guarded.

Reply:
Fixed!